packages/ui/src/components/editor/properties/video/index.tsx

```diff
@@ -99,7 +105,11 @@ export const VideoNodeEditor: EditorComponentType = (props) => {
       description={t('editor:properties.video.description')}
       icon={<VideoNodeEditor.iconComponent />}
     >
-      <ProgressBar value={5} paused={false} totalTime={100} />
+      <ProgressBar
```

Comment:
progress bare doesnt update since this data is not reactive

Reply:
Yes, that's true, I still left it in as we can obtain the duration of the Clip from this. Ill remove it nonetheless